libpod/options.go

```diff
@@ -358,7 +359,7 @@ func WithCNIPluginDir(dir string) RuntimeOption {
 			return define.ErrRuntimeFinalized
 		}
 
-		rt.config.CNIPluginDir = []string{dir}
+		rt.config.PluginDirs = []string{dir}
```

Comment:
Again, don't like the name here - we have more than one type of plugin now. `NetworkPluginDir`? Actually, that probably doesn't work, there are non-CNI network plugins. Probably needs to be `CNIPluginDir` still.